lib/traject/macros/met_csv.rb

```diff
@@ -25,7 +25,7 @@ def met_thumbnail
 
         return if image_json.blank?
         # Some records e.g. 321624, don't return any results
-        result = image_json['results'].first
+        result = Array(image_json['results']).first
```

Comment:
Can you explain where you are seeing the json that doesn't have a `results` entry?

Reply:
I was getting a syntax error here when the array was nil?... I'll have to dive back into that.

Reply:
Can you revert that and see how you got that error?

Reply:
Yeah. Working on it now.